### ISSUE TITLE
Significant performance gap between the JAX and PyTorch models

### ISSUE THREAD

**Issue by JinmingM** [huggingface/lerobot#2598]
Hi, I observed a significant performance gap between the JAX and PyTorch models.

When training with the same real-world dataset and the same data normalization, the JAX version achieves good performance, but the PyTorch version performs very poorly.

I followed the training pipeline described in the README (including checkpoint covert, data loading, normalization, and all default hyperparameters).

Could you please help check whether there are known inconsistencies between the JAX and PyTorch implementations, or if there are additional steps required to reproduce the JAX results using PyTorch?

Thanks!

**Comment by Hegai** [comment]
@JinmingM how did you solve the mismatch in performance?
